generator_test.go

```diff
@@ -376,6 +378,103 @@ func testNewV5DifferentNamespaces(t *testing.T) {
 	}
 }
 
+func Test_defaultHWAddrFunc(t *testing.T) {
+	tests := []struct {
+		n  string
+		fn func() ([]net.Interface, error)
+		hw net.HardwareAddr
+		e  string
+	}{
+		{
+			n: "error",
+			fn: func() ([]net.Interface, error) {
+				return nil, errors.New("controlled failure")
+			},
+			e: "controlled failure",
+		},
+		{
+			n: "no_valid_hwaddr_returned",
+			fn: func() ([]net.Interface, error) {
+				s := []net.Interface{
+					{
+						Index:        1,
+						MTU:          1500,
+						Name:         "test0",
+						HardwareAddr: net.HardwareAddr{1, 2, 3, 4},
+					},
+					{
+						Index:        2,
+						MTU:          1500,
+						Name:         "lo0",
+						HardwareAddr: net.HardwareAddr{5, 6, 7, 8},
+					},
+				}
+
+				return s, nil
+			},
+			e: "uuid: no HW address found",
+		},
+		{
+			n: "valid_hwaddr_returned",
```

Comment:
`snake_case` vs. `PascalCase` once again.

Reply:
Ugh. I didn't notice that in the original PR and would have asked for a change. That's against my personal style. Would anyone be against me going through and retrofitting the tests?

Reply:
Although that seems extremely heavy handed. My personal style ended up being `snake_case` to  improve readability of test outputs.

Reply:
My personal style is `PascalCase` and it was me who converted most of the tests to their current form, back when we got rid of gocheck, so I guess you can blame me! But if you feel strongly about `snake_case`, I can live with that too, so long as we have consistency. If you want to change all the subtests to `snake_case`, go ahead.

Reply:
I'm indifferent FWIW.